doc/api/process.md

```diff
@@ -1626,6 +1626,29 @@ if (process.getuid && process.setuid) {
 or Android).
 
 
+## process.shouldAbortOnUncaughtException
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+The `process.shouldAbortOnUncaughtException` switch controls the behavior
+when the `--abort-on-uncaught-exception` flag was passed on the command line
+or set through [`v8.setFlagsFromString()`][]:
+
+If the flag was passed to Node and `process.shouldAbortOnUncaughtException`
```

Comment:
I assume the default value is `true`. But it is not clear from the documentation.

Reply:
Done